README.md

```diff
@@ -46,16 +49,19 @@ There are two types of management connectivity for NOS VMs: _pass-through_ and _
 _Pass-through management_ interfaces allows the use of the assigned management IP within the NOS VM, management traffic is transparently passed through to the VM, and the NOS configuration can accurately reflect the management IP. However, it is no longer possible to send or receive traffic directly in the vrnetlab container (e.g. for installing additional packages within the container), other than to pre-defined exceptions, such as the QEMU serial port on TCP port 5000.
 
 NOSes defaulting to _pass-through_ management interfaces are:
-- All vJunos routers
+
+* All vJunos routers
```

Comment:
The recent changes made this no longer true -- everything is host-forwarded by default!

Reply:
hopefully done right in [8d977ba](https://github.com/hellt/vrnetlab/pull/287/commits/8d977ba365bc6db73cd1cb1c3506617948ac699b)